packages/office-ui-fabric-react/src/components/DetailsList/examples/DetailsList.Grouped.Example.tsx

```diff
@@ -24,7 +37,14 @@ const _columns = [
     maxWidth: 200
   }
 ];
-const _items = [
+// NOTE: If changing these, also change the initial definition of state.groups below
+const _initialRedNdx = 0;
```

Comment:
Just curious, is there a reason for the difference in naming between `_initialRedNdx` and `_initialGreenIndex`?

Reply:
...maybe I got distracted partway through naming the variables? :P (I'll fix it)